packages/ckeditor5-dev-tests/lib/bin/create-mgit-json.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+module.exports = function createMgitJson( packageJson ) {
+	const mgitJson = {
+		dependencies: {}
+	};
+
+	const dependencies = Object.assign( {}, packageJson.dependencies, packageJson.devDependencies );
+
+	if ( !Object.keys( dependencies ).length ) {
+		return null;
```

Comment:
Why just don't return the `mgitJson` variable? An empty object (`dependencies`) is still valid configuration for Mgit.